go/arrow/flight/flightsql/client_test.go

```diff
@@ -60,6 +60,18 @@ func (m *FlightServiceClientMock) AuthenticateBasicToken(_ context.Context, user
 	return args.Get(0).(context.Context), args.Error(1)
 }
 
+func (m *FlightServiceClientMock) CancelFlightInfo(ctx context.Context, info *FlightInfo, opts ...grpc.CallOption) (CancelFlightInfoResult, error) {
+	panic("not implemented") // TODO: Implement
```

Comment:
can this just follow the pattern above of `args := m.Called(info, opts)` and `return args.Get(0).(CancelFlightInfoResult), args.Error(1)` so we can use the mock to create the tests?

Reply:
OK. I've changed to use mock and added tests of them.